### PR TITLE
replaced args with constructor arguments tuple [CI integration]

### PR DESCRIPTION
**Pull request by CosminNechifor**
I have removed the *args in deploy_contract_web3 method.

### CODE REVIEW

**Comment by LefterisJP** [comment]
note for the future for commit messages: 

Please keep titles within 50 chars. You can add the CI directives after a blank line in the body of the commit message. Guidelines are in the contributing doc found [here](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md#commiting-rules)

**Comment by LefterisJP** [comment]
Why are the API tests failing and why are there 2 new integration tests failing? Is it only in this PR or in general?

**Comment by palango** [comment]
The API tests are having problems with the changed channel identifier. Not sure about the integration tests

**Comment by LefterisJP** [comment]
I am merging this as the failed tests are going to be handled in other PRs.

File: raiden/tests/utils/smartcontracts.py [hunk 1]
```diff
@@ -54,7 +56,7 @@ def deploy_contract_web3(
         contract_name: str,
         deploy_client: JSONRPCClient,
         num_confirmations: int = None,
-        *args,
+        constructor_arguments: typing.Tuple[typing.Any] = (),
```
Comment (hackaugusto) [inline raiden/tests/utils/smartcontracts.py]: this type is not correct, the tuple is not necessarily of length `1`. The correct syntax is [`typing.Tuple[typing.Any, ...]`](https://mypy.readthedocs.io/en/latest/builtin_types.html?highlight=tuple#built-in-types)

**Pull request merged**